main.tf

```diff
@@ -220,3 +220,9 @@ resource "google_storage_bucket_object" "folders" {
   name     = "${each.value.folder}/" # Declaring an object with a trailing '/' creates a directory
   content  = "foo"                   # Note that the content string isn't actually used, but is only there since the resource requires it
 }
+
+resource "google_storage_hmac_key" "hmac_keys" {
```

Comment:
Lets set project explicitly

Reply:
Do you mean to add the next string?
project = var.project_id

Reply:
Yes, otherwise it will depend on provider default

Reply:
Done